### PR TITLE
Add exception trace in response body only in debug mode

### PR DESCRIPTION
**Pull request by neeckeloo**

### CODE REVIEW

**Comment by BigZ** [comment]
Hello @neeckeloo and thanks a lot for your contribution !
Can you tell me more on the context that motivates it ?
On the one hand it's great not to display internal errors to the public, but on the other hand this error serializer can be used to display other errors than 500 such as 400 with details (like fields missing in a form) or 404.

I would suggest to go for something like: 
 - if we're in debug show the full errors anyway
- else if we're not in debug and the error code is < 500 show the full error
- else if we're not in debug and the error code is 500 show an "Internal Server Error" error

File: Subscriber/ExceptionSubscriber.php [hunk 1]
```diff
@@ -16,39 +18,51 @@
...
 
         $this->logger->log('error', $exception->getMessage());
 
         $response = new Response();
-        $response->setContent(json_encode(['errors' => $this->getErrorBody($exception)]));
         $response->setStatusCode(Response::HTTP_INTERNAL_SERVER_ERROR);
 
+        if ($this->kernel->isDebug()
+            || (
+                ($exception instanceof HttpExceptionInterface || $exception instanceof HttpException)
```
Comment (BigZ) [inline Subscriber/ExceptionSubscriber.php]: we should also take into account multi part exceptions

Comment (neeckeloo) [reply Subscriber/ExceptionSubscriber.php]: `Wizards\RestBundle\Exception\MultiPartHttpException` extends `WizardsRest\Exception\HttpException`.

**Comment by BigZ** [comment]
closing in favor of https://github.com/wizardstechnologies/rest-api-bundle/pull/25/files

**Pull request closed without merging**